modules/pipelines-document-attributes.adoc

```diff
@@ -9,4 +9,5 @@
 //
 :pipelines-title: Red Hat OpenShift Pipelines
 :pipelines-shortname: Pipelines
-:pipelines-ver: release-tech-preview-3
+:pipelines-ver: pipelines-1.2
```

Comment:
@sounix000 this value should probably be: release-tech-preview-2
For example for the URL Pavel points out, the correct URL is: https://github.com/openshift/pipelines-tutorial/blob/release-tech-preview-2/01_pipeline/01_apply_manifest_task.yaml

Reply:
Done